DFC.App.MatchSkills.Test/Helpers/MockHelpers.cs

```diff
@@ -5,14 +5,25 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using DFC.App.MatchSkills.Application.Session.Interfaces;
+using DFC.App.MatchSkills.Application.Session.Models;
+using DFC.App.MatchSkills.Controllers;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Http;
 using Moq;
 using Moq.Protected;
+using NSubstitute;
+using FluentAssertions.Common;
+using Microsoft.AspNetCore.Mvc;
 
 namespace DFC.App.MatchSkills.Test.Helpers
 {
     public static class MockHelpers
     {
-        public static  Mock<HttpMessageHandler> GetMockMessageHandler(string contentToReturn="{'Id':1,'Value':'1'}", HttpStatusCode statusToReturn=HttpStatusCode.OK)
+        public static string GetCookieName { get; } = ".matchSkill-session";
```

Comment:
Can You reference the Static CookieName from CookieService please

Reply:
Done